src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+body {
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 40px 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  align-self: center;
+  margin: 10px 0 10px 50px;
+}
+
+.nav__lists {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  color: black;
```

Comment:
it's better use hex instead of reserved color names

Reply:
not fixed
https://github.com/mate-academy/layout_moyo-header/pull/3341/files#r1227124051